lxd/storage_btrfs.go

```diff
@@ -412,14 +411,12 @@ func (s *storageBtrfs) StoragePoolMount() (bool, error) {
 			// detection task.
 			return false, nil
 		}
-
 	}
 
-	mountFlags, mountOptions := lxdResolveMountoptions(s.getBtrfsMountOptions())
 	mountFlags |= s.remount
 	err := syscall.Mount(mountSource, poolMntPoint, "btrfs", mountFlags, mountOptions)
 	if err != nil {
-		logger.Errorf("failed to mount BTRFS storage pool \"%s\" onto \"%s\" with mountoptions \"%s\": %s", mountSource, poolMntPoint, mountOptions, err)
+		logger.Errorf("Failed to mount BTRFS storage pool \"%s\" onto \"%s\" with mountoptions \"%s\": %s", mountSource, poolMntPoint, mountOptions, err)
```

Comment:
Can we agree on a standard in terms of logging? :) Full sentences with capitalization and punctuation or all small caps (barring abbreviations) without punctuation. :)